### PR TITLE
BookStore : Add styles

### PR DESCRIPTION
**Pull request by ohyajk**
In this feature, I added some style according to the given design in Zeplin.

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -6,12 +6,16 @@ import { removeBook } from '../redux/books/books';
...
+        <div className="py-2">
+          <button type="button" className="text-[#0290ff] pr-2 cursor-pointer ">Comments</button>
+          <button onClick={() => dispatch(removeBook(id))} type="button" className="text-[#0290ff] pr-2 cursor-pointer ">Remove</button>
+          <button type="button" className="text-[#0290ff] pr-2 cursor-pointer ">Edit</button>
+        </div>
       </div>
-      <button onClick={() => dispatch(removeBook(id))} type="button" className="border-2 border-fuchsia-500 p-2 cursor-pointer rounded-lg">Remove</button>
     </div>
   );
 };
```
Comment (DammyShittu) [inline src/components/Book.jsx]: - [x] We are required to style the app as it is on the design exactly. Kindly take note of things like font weight, font size, font family and so on.

- [x] You are also missing some elements like the user icon on the top right, the elements on the right side of the book components. Kindly try to replicate the design exactly.

  Your Implementation | Zeplin Design
  :---:|:---:
  urit | die

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @ohyajk,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Kindly fix your project title and description on the readme

  utir

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.jsx (hunk 1 above)
Comment (ohyajk) [reply src/components/Book.jsx]: Thank you @DammyShittu I have fixed these issues.

**Review by pinkmoon25 — APPROVED** [review]
Hi @ohyajk , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
linter checks :heavy_check_mark: 
good readme :heavy_check_mark: 
changes implemented :heavy_check_mark: 
routing :heavy_check_mark: 

## Optional suggestions

**[OPTIONAL] You can make the font-size, color and family exactly like the Zepplin design by [visiting this design](https://pedantic-beaver-cb923b.netlify.app/#/) and inspecting it for the exact styles** 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ohyajk** [comment]
Thank you @PinkMoon25 for approval. 😊

**Pull request merged**